packages/patternfly-4/react-table/src/components/Table/Table.md

```diff
@@ -406,6 +406,7 @@ class ContactExpandableTable extends React.Component {
         },
```

Comment:
fwiw, just noticed this class is called `ContactExpandableTable`. Should that be `CompactExpandableTable`?